docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -18,6 +18,8 @@ Note that this command is not supported for the remote client.
 
 Generate files instead of printing to stdout.  The generated files are named {container,pod}-{ID,name}.service and will be placed in the current working directory.
 
+Note: On a system with SELinux enabled, the generated files will inherit contexts from current working directory. Depending on your SELinux setup, you may need to change it using `restorecon`, `chcon`, or `semanage`, to make them work with systemd.
```

Comment:
```suggestion
Note: On a system with SELinux enabled, the generated files will inherit contexts from the current working directory. Depending on your SELinux setup, you may need to change it using `restorecon`, `chcon`, or `semanage`, to make them work with systemd.
```

Reply:
I would also mention that usage of `mv -Z` and `cp -Z` which will take care of that implicitly.